torchaudio/csrc/decoder/bindings/pybind.cpp

```diff
@@ -0,0 +1,140 @@
+#include <torch/extension.h>
+
+#include <torchaudio/csrc/decoder/bindings/_decoder.cpp>
+#include <torchaudio/csrc/decoder/bindings/_dictionary.cpp>
+
+PYBIND11_MODULE(_torchaudio_decoder, m) {
+#ifdef BUILD_CTC_DECODER
```

Comment:
I think we can remove this directive since we are creating a library file separate from `libtorchaudio`.